randovania/games/dread/gui/dialog/dread_cosmetic_patches_dialog.py

```diff
@@ -15,6 +15,9 @@ def __init__(self, parent: QWidget, current: DreadCosmeticPatches):
         self.setupUi(self)
         self._cosmetic_patches = current
 
+        for room_gui_type in DreadRoomGuiType:
+            self.room_names_dropdown.addItem(room_gui_type.value, room_gui_type)
```

Comment:
We don't expose the enum value to the end user: those are internal values for serialization/etc.
If we need to add some pretty name we usually add a property called `long_name`. Check `randovania/games/prime2/layout/translator_configuration.py` or `randovania/layout/base/dock_rando_configuration.py`.

(reason for `long_name` is legacy stuff, but keeping the "rule" helps with generic code)